### PR TITLE
config: pipeline: add initial Tast tests

### PR DESCRIPTION
**Pull request by a-wai**
This change adds a `tast-quickcheck-x86` job, running on 2 Intel-based devices in order to help debug the new Tast implementation.

### CODE REVIEW

**Comment by a-wai** [comment]
So, this PR ends up being a significant rework of the pipeline config as much as it is about adding Tast tests.

As such, the more comments & suggestions about this proposal, the better :)

(and given the scope of changes I've added `staging-skip` in order to ensure I don't break the whole system)

**Comment by nuclearcat** [comment]
Starting staging test

File: config/jobs-chromeos.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        - x86-board
+        - CONFIG_MODULE_COMPRESS=n
+
+jobs:
+
+  baseline-arm64-chromeos-mediatek: &baseline-job
+    template: baseline.jinja2
+    kind: test
+
+  baseline-arm64-chromeos-qualcomm: *baseline-job
```
Comment (pawiecz) [inline config/jobs-chromeos.yaml:230]: I might be missing something: what is the purpose of duplicating job definitions with the same body? Is it for logging/tracking progress? Wouldn't adding `job: baseline-job` in the schedule item definition cut it?

**Review by pawiecz — APPROVED** [review]
LGTM (except Tast-specific bits in second-to-last patch - I don't have any expertise there)

File: config/jobs-chromeos.yaml (hunk 1 above)
Comment (a-wai) [reply config/jobs-chromeos.yaml:230]: That's a good question, maybe @nuclearcat knows?

I confess I kept things as they were previously (1:1 mapping between jobs and scheduler entries), but I'd be happy to drop duplicates if we can actually schedule multiple instances of a given job.

**Comment by a-wai** [comment]
Changed a bit how tast tests are listed, now depends on https://github.com/kernelci/kernelci-core/pull/2405

**Comment by nuclearcat** [comment]
Tested on staging

**Pull request merged**